lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -29,11 +29,11 @@
 * offsetText
 * label
 
-Note that these are separate artists from Axis class of the
-original mpl, thus most of tick-related command in the original mpl
-won't work, although some effort has made to work with. For example,
-color and markerwidth of the ax.axis["bottom"].major_ticks will follow
-those of Axes.xaxis unless explicitly specified.
+Note that these are separate artists from `matplotlib.axis.Axis`, thus
+most of tick-related functions in Matplotlib won't work, although some
+effort has made to work with. For example, color and markerwidth of the
```

Comment:
'although some effort has made to work with' is very ambigous - what is working with what?

Reply:
let's just remove that phrase